integration-cli/docker_cli_rm_test.go

```diff
@@ -19,6 +20,9 @@ func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
 	}
 	defer os.RemoveAll(tempDir)
 
+	out, _, _ := dockerCmdWithStdoutStderr(c, "ps", "-a")
+	fmt.Printf("out: %s", out)
```

Comment:
Right, Log only writes out if there's a failure. I usually put c.Fatal after it to debug stuff